### ISSUE TITLE
Menu items with checkboxes not properly annotated for a11y tools

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#4911]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.230
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
On RStudio Server, main menu items with checkmarks (e.g. Debug / On Error / ..., View / Panes / Show All Panes) do not have markup that allows the checked/unchecked state to be seen by screen readers.

On RStudio Server and Desktop, pop-up menus with checkmarks such as the Workspace Panes toolbar button do not have markup that allows the checked/unchecked state to be seen by screen readers.

### Describe the problem in detail
Screen reader users are unable to tell that these items are checkable, and what the current state is.

Also, for mutually exclusive items (radio-button like, such as those under Debug / On Error, there is no annotation to indicate they behave this way. This is also the case for sighted users, who kinda just have to figure it out.

### Describe the behavior you expected
At a minimum, screen readers need to tell user a given menu item is checkable, and indicate the current checked state, via `role=menuitemcheckbox` and `aria-checked=true`.

It would be nice if there was also indication of mutually exclusive items via `role=menuitemradio`.
